bin/rbenv-ctags

```diff
@@ -17,6 +17,16 @@ generate_ctags_in() {
   (cd "$1"; ctags -R --languages=ruby)
 }
 
+generate_c_ctags_in() {
```

Comment:
Since this method gets called only once, I would rather inline it at the place where it's needed